api/api_test.go

```diff
@@ -44,8 +44,9 @@ func cfg() *config.Config {
 		ServiceAuthToken:         serviceAuthToken,
 		EnablePrivateEndpoints:   enablePrivateEndpoints,
 		MaxDatasetOptions:        200,
+		DefaultMaxLimit:          1000,
 		MongoConfig: config.MongoConfig{
-			Limit:  0,
+			Limit:  20,
```

Comment:
Just so I can understand this change - what does this limit do?

Reply:
This value is the default limit for pagination.
When we do a GET request to an endpoint that can return a list of items, we may only return a subset of the total items. The caller can provide query paramters:
- limit: the maximum number of items that they want to request
- offset: the first index that they want to requrest
if they don't provide a limit, a value  of 20 will be used.
(well, in fact this is the value for testing, but it's the same in the real config)